### PR TITLE
[AI Gallery] Add quantized LLMs with Ollama

### PR DESCRIPTION
**Pull request by romilbhardwaj**
Adds an example showing how quantized LLMs can be run on CPUs (and even the user's local machine) with Ollama.

### CODE REVIEW

**Review by Michaelvll — APPROVED** [review]
Thanks for adding this @romilbhardwaj! LGTM.

File: llm/ollama/README.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
+# Ollama: Run quantized LLMs on CPUs
+<p align="center">
+  <img src="https://i.imgur.com/HfqnGVA.png" width="400">
+</p>
+
+[Ollama](https://github.com/ollama/ollama) is popular library for running LLMs on both CPUs and GPUs. 
+It supports a wide range of models, including quantized versions of `llama2`, `llama2:70b`, `mistral`, `phi`, `gemma:7b` and many [more](https://ollama.com/library). 
+You can use SkyPilot to run these models on CPU instances on any cloud provider, Kubernetes cluster, or even on your local machine.
```
[the remark this answers is not in the document]
Comment (romilbhardwaj) [reply llm/ollama/README.md:8]: Good point - I've updated the guide to have an explicit optional section on how to run locally. Does that help?

File: docs/source/_gallery_original/index.rst [hunk 2]
```diff
@@ -23,6 +23,7 @@ Contents
 
    vLLM <frameworks/vllm>
    Hugging Face TGI <frameworks/tgi>
+   Ollama <frameworks/ollama>
```
Comment (concretevitamin) [inline docs/source/_gallery_original/index.rst:26]: I found it a bit odd to group it under "Inference Engines". How about we rename to "Inference Frameworks"?

File: llm/ollama/README.md [hunk 3]
```diff
@@ -0,0 +1,296 @@
+# Ollama: Run quantized LLMs on CPUs and GPUs
+<p align="center">
+  <img src="https://i.imgur.com/HfqnGVA.png" width="400">
+</p>
+
+[Ollama](https://github.com/ollama/ollama) is popular library for running LLMs on both CPUs and GPUs. 
+It supports a wide range of models, including quantized versions of `llama2`, `llama2:70b`, `mistral`, `phi`, `gemma:7b` and many [more](https://ollama.com/library). 
+You can use SkyPilot to run these models on CPU instances on any cloud provider, Kubernetes cluster, or even on your local machine. 
+
```
Comment (concretevitamin) [inline llm/ollama/README.md:9]: Shall we add a quick pointer here something like: (Of course, GPUs are supported too if you have them.)

Should we add a quick section to the end of this doc on how to enable GPUs? Reason is a reader currently sees only CPU mentioned in this doc.

**Comment by concretevitamin** [comment]
Can we add a new button too? https://github.com/skypilot-org/skypilot/blob/master/docs/source/_static/custom.js#L27-L31

File: llm/ollama/README.md (hunk 3 above)
Comment (romilbhardwaj) [reply llm/ollama/README.md:10]: Good call - added pointers for GPUs

**Pull request merged**